README.md

```diff
@@ -157,22 +157,28 @@ Also we have a lot of auth specific env variables:
 | `AUTH_AUTH0_HOST`         | No                   | Auth0 Host                                                          | Any string       |                |
 | `AUTH_AUTH0_NAME`         | No                   | Auth0 Name                                                          | Any string       |                |
 | `AUTH_AUTH0_SECRET`       | No                   | Auth0 Secret                                                        | Any string       |                |
+| `AUTH_AUTH0_SCOPE`        | No                   | Auth0 Scope                                                         | Any string       | `openid email profile offline_access` |
 | `AUTH_AZURE_AD_CLIENT_ID` | No                   | Azure AD Client ID                                                  | Any string       |                |
 | `AUTH_AZURE_AD_NAME`      | No                   | Azure AD Name                                                       | Any string       |                |
 | `AUTH_AZURE_AD_SECRET`    | No                   | Azure AD Secret                                                     | Any string       |                |
 | `AUTH_AZURE_AD_TENANT_ID` | No                   | Azure AD Tenant ID                                                  | Any string       |                |
+| `AUTH_AZURE_AD_SCOPE`| No                 | Azure AD Scope                                                      | Any string       | `openid profile user.Read email offline_access` |
```

Comment:
please fix formatting

Reply:
@akryvasheyeu , fixed. Result formation is fine anyway
![image](https://github.com/epam/ai-dial-chat/assets/20163971/f497feae-922c-4255-90ec-a2621f80e014)